scripts/cover.sh

```diff
@@ -40,6 +40,6 @@ make munit-opae-c munit-opae-c-events
 
 lcov --directory . --zerocounters
 lcov -c -i -d . -o coverage.base
-./bin/munit-opae-c --gtest_filter="enum*:buffer*:*properties*:sysfs*:mmio*"
-./bin/munit-opae-c-events
 
+./bin/munit-opae-c --gtest_filter="enum*:buffer*:*properties*:sysfs*"
```

Comment:
Why are we excluding mmio tests here?

Reply:
It's only excluded for this branch. It will be included in the later PRs.